src/xcode/ENA/ENA/Source/Scenes/Home/HomeViewController.swift

```diff
@@ -151,7 +151,18 @@ final class HomeViewController: UIViewController, RequiresAppDependencies {
 				supportedCountries = []
 			}
 			
-			DispatchQueue.main.asyncAfter(deadline: .now() + 0.5) {
+			// As per feature requirement, the delta onboarding should appear with a slight delay of 0.5
+			var delay = 0.5
+			
+			#if DEBUG
+			if isUITesting {
+				// In UI Testing we need to increase the delaye slightly again.
+				// Otherwise UI Tests fail
+				delay = 1.5
```

Comment:
this smells like a flaky UI Test

Reply:
This behavior never happens in reality. This popup only appears when the user opens the app the first time. But as the unit test always run as initial this alert shows up in the UI test and we have to dismiss this first. This worked until the last upgrade of XCode I think. Now I was facing this issue had to resolve it.